packages/react-native-uifabric/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "react-native-uifabric",
+  "version": "0.12.10",
+  "description": "A react-native companion to fabric web that only depends on stock controls and theming",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "private": true,
+  "scripts": {
+    "build": "just-scripts build --useMetro",
+    "just": "just-scripts",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "lint": "just-scripts eslint",
+    "start": "just-scripts dev",
+    "start-test": "just-scripts jest-watch",
+    "update-snapshots": "just-scripts jest -u",
+    "verify-api": "just-scripts verify-api-extractor",
+    "update-api": "just-scripts update-api-extractor",
+    "run-win32": "rex-win32 --bundle RNTester.win32 --component RNTesterApp --basePath ./dist/win32/dev",
```

Comment:
Are the rex-win32 dependencies correct?

Reply:
no, Rex is not working yet, because bundling is not implemented. I'll fix it when I got bundling task working.